nav2_experimental/nav2_rl/nav2_turtlebot3_rl/navigation_task_env_test.py

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from navigation_task_env import NavigationTaskEnv
+
+class Test(NavigationTaskEnv):
+    '''
+    This is a simple test to check if the navigation task is
+    getting the path using the ComputePathToPose action.
+    '''
+    def __init__(self):
+        super().__init__()
+
+    def get_actions(self):
+        pass
+
+    def main(self, args=None):
```

Comment:
Can we rewrite the test as a py.test unit test?

Reply:
I think I want to wait before we make the test, so that we know what all we want there. This was just a simple way to see if I was getting the path correctly, but we might want more.